issueWork/UI/UX__issue/README.md

```diff
@@ -0,0 +1,27 @@
+# UI / UX issue  
+
+## Changes made:  
+
+> 1. Complete overhall of webpage layout based on previously received feedback. Essentially creating a clean and somewhat minimal style.
+ - ### Laws:  
+   - **Aesthetic-Usability**
+   - **Jakob's Law**
+   - **Hick's Law**
+> 2. Attached and created an accordin to the hero section's breif about me. Which when clicked expands to a larger more indepth about me section.  
+ - ### Laws:  
+   - **Law of Proximity**
+   - **Doherty Threshold (currently 300ms)**
+   - **Law of Common Region**
+
+> 3. Created a Projects section. Currently grouping three mock projects with title, description, and what poblem it solved. May introduce languages and technology used to create project. This is dependant on feedback received.  
+ -  ### Laws:    
+    - **Law of Similarity**
+    - **Tesler's Law**
+
+> 4. Though not completed the contact form with be a modal with only 3 inputs. 
+ - ### Law:  
+   - **Postel's Law**
+
+> 5.  In the final implementation the nav will be affixed to the top of the page. Idea here is to utilize Miller's Law in a way that the viewer is only seeing the nav and three projects. This way upon review of projects the contact me button is not out of sight.
+- ### Law:  
+   - **Miller's Law**
```

Comment:
I kinda see where you're going with Miller's Law here. I don't know if it completely fits. Though I do see that you're trying to keep that nav simple, which I do think is good design.

Reply:
Right and with the current way its setup I tend to thing the about and project links are not needed and may remove them.